internal/compiler/program_test.go

```diff
@@ -312,3 +312,46 @@ func BenchmarkNewProgram(b *testing.B) {
 		}
 	})
 }
+
+func TestMissingReferenceFile(t *testing.T) {
```

Comment:
No need for this; if you fixed the bug, the `declarationEmitIndexTypeNotFound` test should have been updated when you ran the tests.

Reply:
Just push to this one, please